src/workers/personalizedDigestEmail.ts

```diff
@@ -166,6 +201,18 @@ const worker: Worker = {
       return;
     }
 
+    const growthbookClient = getUserGrowthBookInstace(user.id, {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      trackingCallback: (experiment, result) => {
+        // TODO notify allocation
+      },
```

Comment:
I have to test locally that callback is triggered. Need to setup some dev experiment.

Reply:
I'd suggest putting the tracking callback in `getUserGrowthBookInstace` to make it fault tolerant for future uses as well